src/sundials/sundials_logger_impl.h

```diff
@@ -59,4 +60,37 @@ struct SUNLogger_
   SUNErrCode (*destroy)(SUNLogger* logger);
 };
 
+static void sunCreateLogMessage(SUNLogLevel lvl, int rank, const char* scope,
```

Comment:
Does this need to be define in the header?

Reply:
Need to be - no, but it doesn't hurt for this utility I don't think.

Reply:
It doesn't hurt but we generally don't define things in a header unless they need to be there (i.e., to inline) here and this was in a `.c` file before

Reply:
Oops, I read this as `inline` already. It would need `inline` to stay in the header. Ill move it.